lib/eslint.js

```diff
@@ -757,6 +757,8 @@ module.exports = (function() {
      */
     api.verify = function(textOrSourceCode, config, filenameOrOptions, saveState) {
         const text = (typeof textOrSourceCode === "string") ? textOrSourceCode : null;
+        const selectors = new Set();
```

Comment:
Actually it looks like `emitter.eventNames()` was only added in Node 6.0.0, so we can't use it.